src/scss/components/_modal-box.scss

```diff
@@ -1,5 +1,17 @@
 @use "../common/forwarded" as helpers;
 
+:root {
+	--stop-scroll-indent-right: 0;
```

Comment:
I would name it somthing like `--width-of-scrollbar` because it is a with of scrollbar

Reply:
Done